corehq/apps/app_manager/migrations/0003_legacy_child_modules.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq import toggles
+from corehq.apps.app_manager.dbaccessors import get_app_ids_in_domain, get_current_app
+from corehq.util.log import with_progress_bar
+
+
+def flag_legacy_child_module_domains(*args, **kwargs):
+    """Enable the LEGACY_CHILD_MODULES flag for domains that need it"""
+    domains = set(toggles.BASIC_CHILD_MODULE.get_enabled_domains() +
```

Comment:
Should this also include apps that use custom case tiles, even if they don't currently use child modules, in case they turn child modules on in the future?

Reply:
I don't think so - the two aren't incompatible, but it's not always backwards compatible.  If a project starts using child modules later, they'll only have an issue when they make a new child module which changes the module indices.  That's similar to adding a normal new module or otherwise changing the ordering, and I think its appropriate to leave it to them to address.

Reply:
cool, thanks for explaining